examples/example_integration.cpp

```diff
@@ -123,6 +128,30 @@ int main() {
       // Close an opened view
       ct.views().closeViewWithID(viewID);
     } break;
+    case 12: {
```

Comment:
with case "12" add a breadcrumb entry with the current timestamp, don't record any crash.